src/Util.php

```diff
@@ -230,4 +231,22 @@ public static function getShopsTableForeignKey(): string
     {
         return Str::singular(self::getShopsTable()).'_id';
     }
+
+    /**
+     * Checking to see if you need to use the native App Bridge
+     *
+     * @return bool
+     */
+    public static function useNativeAppBridge(): bool
+    {
+        $currentFrontendEngine = self::getShopifyConfig('frontend_engine') ?? FrontendEngine::BLADE;
+
+        switch ($currentFrontendEngine) {
```

Comment:
@enmaboya Switch is probably not needed since its only React we need to look for.

```return !$currentFrontendEngine->isSame(FrontendEngine::REACT);```

So "if current engine is not React, return true and use app bridge"... React will return false and not use app bridge.